src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -144,7 +146,15 @@ export class LoadoutBuilder extends React.Component<Props & UIViewInjectedProps,
         Intellect: { min: 0, max: 10, ignored: false },
         Strength: { min: 0, max: 10, ignored: false }
       },
-      lockedSeasonalMods: [],
+      lockedArmor2Mods: {
+        [ModPickerCategories.general]: new Array<LockedArmor2Mod>(),
```

Comment:
I swear there is a better way to do this but I can't figure it out.

Reply:
99% sure you can just do `[]`

Reply:
You are right